Makefile

```diff
@@ -26,6 +26,7 @@ install:
 	@echo "Installing cron job..."
 	@install -Dm0644 src/recap.cron $(DESTDIR)$(CRONDIR)/recap
 	@echo "Installing docs..."
+	@install -dm0755 $(DESTDIR)$(DOCDIR)/recap
```

Comment:
@jamrok Thanks for reporting this, I was able to reproduce the error:

```bash
root@docker:/recap# cat /etc/os-release; uname -r
NAME="Ubuntu"
VERSION="14.04.5 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.5 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
3.10.0-514.10.2.el7.x86_64

root@docker:/recap# make install
Installing scripts...
Installing man pages...
Installing configuration...
Installing cron job...
Installing docs...
install: failed to access '/usr/local/share/doc/recap': No such file or directory
make: *** [install] Error 1
```

Digging around I found the problem is the underlying directory that does not exist:

```bash
root@docker:/recap# ls -ld /usr/local/share/doc
ls: cannot access /usr/local/share/doc: No such file or directory

root@docker:/recap# touch TEST
root@docker:/recap# install -D TEST -t /usr/local/share/doc/recap 
install: failed to access '/usr/local/share/doc/recap': No such file or directory
```

Comparing the manual of `install` between 14.04 and 16.06 I see differences:

```bash
## 14.04
root@docker:/recap# man install | grep -- -D
       -D     create all leading components of DEST except the last, then copy SOURCE to DEST

## 16.06
root@docker:~# man install | grep --  -D
       -D     create all leading components of DEST except the last, or all components of --target-directory, then copy SOURCE to DEST
```

Your patch should solve this problem on `14.04` but interestingly enough there is no package in the path in question: `/usr/local/share/doc/`

```bash
root@docker:/recap# apt-file search /usr/local/share/doc/
root@docker:/recap#
```

Nor mention of it in the  [LFH](http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/usr.html).

---

Even when this might not be a problem in newer Ubuntu versions or other distros it does not hurt having the extra line to create it. I'm good with the proposed change.

:+1: